cluster/addons/cluster-monitoring/standalone/heapster-controller.yaml

```diff
@@ -7,26 +7,23 @@
 apiVersion: v1
 kind: ReplicationController
 metadata:
-  name: heapster-v15
+  name: heapster-v1.0.0-beta1
   namespace: kube-system
   labels:
     k8s-app: heapster
-    version: v15
```

Comment:
I looked at it in a little more detail, and it looks like this should be fine because the addon-updater does a `stop` followed by a `create` rather than a rolling-update.